Makefile

```diff
@@ -116,7 +116,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 	$(KUSTOMIZE) build config/crd | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
 .PHONY: deploy
-deploy: install manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
Can you check but I think `install` can also be removed from other targets, like `local-deploy-with-watcher`?